main.py

```diff
@@ -409,6 +432,27 @@ def add_metadata(self, circuit_id):
         evc.sync()
         return jsonify("Operation successful"), 201
 
+    @rest("v2/evc/metadata/<key>", methods=["DELETE"])
+    @validate(spec)
+    def bulk_delete_metadata(self, data, key):
+        """Delete metada from a bulk of EVCs"""
+        circuit_ids = data.pop("circuit_ids", None)
+        if circuit_ids is None:
+            raise BadRequest("The key 'circuit_ids' is missing.")
```

Comment:
Related to my prior comment, let's also add a `requestBody` to the delete endpoint that you're adding reusing the same component. Typically, openapi spec 3.0 detects it as an error in the spec, but since we're not generating auto clients, and if our openapi-core validation works with it it's also fair game since we're already a boy in a DELETE endpoint anyway. Could you double check if our `@validate` decorator would work for the delete endpoint with `requestBody`? If it doesn't work, then we can keep the spec of this endpoint without documenting the `requestBody`.

Reply:
Yes it works. Added

Reply:
Cool. Fantastic to know that it also works.